metrics/write.go

```diff
@@ -109,7 +109,7 @@ func cloneRequest(r *http.Request) *http.Request {
 }
 
 func (c *Client) write() error {
```

Comment:
Maybe refactor for later, but the write method fails after 20 remote write errors i.e 20 ticks, I wonder if we can just fail early, or log each error as they happen, as your requestInterval might be large and you might need to wait.

Reply:
Do you mind adding an issue? I didn't mean to change client here

Reply:
Sure will add!

Reply:
https://github.com/prometheus-community/avalanche/issues/81